### DISCUSSION TITLE
Clear user generated thumbnails button

### DISCUSSION THREAD

**Discussion by emrahnazif** [directus/directus#16019] [Feature Requests]
Depending on the use case, thumbnails can serve for a while and occupies disk spaces for no reason. 
It could be to nice idea to have a button to clear all user/system generated thumbnails images maybe with some conditional rules (remove all thumbnails older than 1 month etc).

This idea should be investigated with the https://github.com/directus/directus/discussions/13559 and https://github.com/directus/directus/issues/14796 as well.

**Comment by azrikahar** [comment]
Recently #15977 added support for Flows Manual Trigger buttons to be clickable without making any selection. Maybe this manual trigger can also be added to Files (and Users) module's sidebar so that it'll not only make this "clear user generated thumbnails" action possible, but also opens the door to many other use cases.

That being said, the CRUD operations in Flows will also need to first have official support for system collections as mentioned in https://github.com/directus/directus/discussions/14362#discussioncomment-3121079.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
